Misc/NEWS.d/next/Core and Builtins/2019-07-20-22-34-42.bpo-37444.UOd3Xs.rst

```diff
@@ -0,0 +1 @@
+Update differing exception between buildints and importlib
```

Comment:
Corrects spelling of builtins, specify dunder methods, and add inline markup
```suggestion
Update differing exception between :meth:`builtins.__import__()` and
:mod:`importlib.__import__()`.
```

Reply:
You  actually don't need the parentheses when you specify `:meth:`; Sphinx will add them for you.

Reply:
Oh good to know, thanks. I updated the suggestion accordingly.

Reply:
Thanks, I've updated it